app/aws/s3.py

```diff
@@ -60,8 +60,10 @@ def get_job_metadata_from_s3(service_id, job_id):
     return obj.get()["Metadata"]
 
 
-def remove_job_from_s3(service_id, job_id):
-    return remove_s3_object(*get_job_location(service_id, job_id))
+def remove_job_batch_from_s3(jobs):
+    bucket = resource("s3").Bucket(current_app.config["CSV_UPLOAD_BUCKET_NAME"])
+    object_keys = [FILE_LOCATION_STRUCTURE.format(job.service_id, job.id) for job in jobs]
+    bucket.delete_objects(Delete={"Objects": [{"Key": key} for key in object_keys]})
```

Comment:
Is there a limit on the number of keys we can provide to s3 for mass deletion?

Reply:
There is, we can delete [up to 1000 objects per call](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/delete_objects.html#S3.Client.delete_objects)

Reply:
yeah this is what I remember.. I am wondering because a caller could call this function and not be aware of the limit. It should have some documentation at a minimum to describe the limit, or have a loop to iterate through it.

Reply:
added a loop to `remove_jobs_from_s3()` to prevent us from going over 1000 AWS limit as well as a new test for this function.